src/test/java/uk/gov/companieshouse/model/validator/CompanyNumberValidatorTest.java

```diff
@@ -11,10 +11,7 @@
 import java.util.stream.Stream;
 
 import org.junit.jupiter.api.Test;
-import org.junit.jupiter.api.extension.ExtendWith;
-import org.springframework.test.context.junit.jupiter.SpringExtension;
 
-@ExtendWith(SpringExtension.class)
 public class CompanyNumberValidatorTest {
```

Comment:
Remove **public** from class declaration. Also, we use the **@DisplayName** annotation on test methods to give a more verbose way of describing a test. Can this annotation be added to test methods in the class please.

Reply:
Done in new commit